pkg/deploy/controller/deploymentconfig/factory.go

```diff
@@ -40,7 +40,7 @@ func (factory *DeploymentConfigControllerFactory) Create() controller.RunnableCo
 		},
 	}
 	queue := cache.NewFIFO(cache.MetaNamespaceKeyFunc)
-	cache.NewReflector(deploymentConfigLW, &deployapi.DeploymentConfig{}, queue, 2*time.Minute).Run()
+	cache.NewReflector(deploymentConfigLW, &deployapi.DeploymentConfig{}, queue, 30*time.Second).Run()
```

Comment:
Hrm - I guess the next rebase (after this) we'll pull the list vs sync separation.  I'm concerned a bit that this will regress us a bit in performance, and whether we'll have to do a lot of work to pick that up.

@rrati do your changes to resync/relist depend on Informers, or can they work for any reflector?